source/_posts/2016-09-30-spark-streaming-on-yarn.markdown

```diff
@@ -0,0 +1,324 @@
+---
+layout: post
+title: "Long running Spark Streaming jobs on YARN cluster"
+date: 2016-10-03
+comments: true
+categories: [spark, yarn, hdfs]
+---
+
+Long running Spark Streaming job, once submitted to the YARN cluster should run forever until it will be intentionally stopped.
```

Comment:
*A* long-running Spark Streaming job ... to *a* YARN cluster... until it is (present)